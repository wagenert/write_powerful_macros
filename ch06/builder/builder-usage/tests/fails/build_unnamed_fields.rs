use builder_macro::Builder;

#[derive(Builder)]
struct UnnamedStruct(i32, String);

fn main() {}
