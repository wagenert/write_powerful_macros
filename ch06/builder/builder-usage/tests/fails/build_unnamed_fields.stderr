error: proc-macro derive panicked
 --> tests/fails/build_unnamed_fields.rs:3:10
  |
3 | #[derive(Builder)]
  |          ^^^^^^^
  |
  = help: message: Only structs with named fields are supported
